lib/shared-method.js

```diff
@@ -137,7 +137,9 @@ function SharedMethod(fn, name, sc, options) {
     this.errors = [this.errors];
   }
 
-  this.stringName = (sc ? sc.name : '') + (isStatic ? '.' : '.prototype.') + name;
+  var addPrototypeString = !/^prototype\./.test(name);
+  this.stringName = (sc ? sc.name : '') +
+    (!isStatic && addPrototypeString ? '.prototype.' : '.') + name;
```

Comment:
Is this based on my earlier [comment](https://github.com/strongloop/strong-remoting/pull/275#issuecomment-176144736)?

> The code for creating (defining) new shared methods does not support "prototype.name" notation. I am not sure if this is something to change too. Can we discuss it outside of this pull request please?